src/Kdyby/Doctrine/Diagnostics/Panel.php

```diff
@@ -388,8 +388,7 @@ public static function renderException($e, Nette\DI\Container $dic)
 
 			if ($invalidTable = Strings::match($e->getMessage(), '~table \'(.*?)\'~i')) {
 				foreach ($em->getMetadataFactory()->getAllMetadata() as $class) {
-					/** @var Kdyby\Doctrine\Mapping\ClassMetadata $class */
-					if ($class->getTableName() === $invalidTable[1]) {
+					if ($class instanceof Doctrine\ORM\Mapping\ClassMetadata && $class->getTableName() === $invalidTable[1]) {
```

Comment:
I don't understand why phpstan wants this. `Kdyby\Doctrine\Mapping\ClassMetadata` extends `Doctrine\ORM\Mapping\ClassMetadata` so the method should exist, right? Do you know what's up here?

Reply:
phpstan 0.9 does not want to work with doctype $class